waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -1159,14 +1176,16 @@
         service-id->failed-instances-transient-store (atom {})
         replicaset-spec-builder-ctx (assoc replicaset-spec-builder
                                       :log-bucket-sync-secs log-bucket-sync-secs
-                                      :log-bucket-url log-bucket-url)
+                                      :log-bucket-url log-bucket-url
+                                      :reverse-proxy-flag "GRPC_TRANSCODER")
         replicaset-spec-builder-fn (let [f (-> replicaset-spec-builder
                                                :factory-fn
                                                utils/resolve-symbol
                                                deref)]
                                      (assert (fn? f) "ReplicaSet spec function must be a Clojure fn")
                                      (fn [scheduler service-id service-description]
-                                       (f scheduler service-id service-description replicaset-spec-builder-ctx)))
+                                       (let [scheduler' (assoc scheduler :reverse-proxy-flag "GRPC_TRANSCODER")]
```

Comment:
`GRPC_TRANSCODER` value should come from the scheduler, e.g. called `reverse-proxy-flag`?

Reply:
We should not be `assoc`-ing to create `scheduler'`.
`scheduler` already has `reverse-proxy-flag`.